### PR TITLE
feat: add warnet api service

### PR DESCRIPTION
**Pull request by Extheoisah**
Todo(in a follow up pr):
    - add auth
    - add middleware
    - define network properties

### CODE REVIEW

File: api-compose.yml [hunk 1]
```diff
@@ -0,0 +1,32 @@
+version: "3"
+services:
+  db:
+    image: postgres:14.0
+    ports:
+      - "5432:5432"
+    environment:
+      POSTGRES_PASSWORD: password
```
Comment (okjodom) [inline api-compose.yml:8]: there's a number of config and app secrets committed in code, and we should consider secure provisioning

File: api/main.py [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+    version="0.0.1",
+)
+
+@app.get("/")
+def read_root():
+    return {"Hello Viking": "Welcome to Warnet!"}
+
+@app.post("/warnet")
+def read_root():
+    return {"Hello": "warnet"}
```
Comment (okjodom) [inline api/main.py]: ❔ : it's unclear to me how we'd execute warnet automated docker functions from this api which is also hosted within a docker environment

**Review by okjodom — COMMENTED** [review]
tACK. Works as proposed.
Needs clarification o how a docker wrapped api can run `warnet`

File: api-compose.yml (hunk 1 above)
Comment (Extheoisah) [reply api-compose.yml:8]: Agree. This is for local dev. We're hardcoding some secrets in others parts of the code too. I'll create an issue for it

**Pull request merged**